.chloggen/replication-stats-conversion-failure.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: postgresqlreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Deprecation of replication lag metrics in favor of more precise 'write_ms','replay_ms' and 'flush_ms'"
```

Comment:
Can we update this to specifically reference the metrics which is being deprecated (`postgresql.wal.lag`) and the one we're adding to replace it (`postgresql.wal.lag`)?